tools/windows-node-installer/README.md

```diff
@@ -30,6 +31,10 @@ destroy a Windows instance on the selected provider. To create an instance, `wni
 requires extra information such as image id and instance type. Some optional flags include directory path to 
 windows-node-installer.json file and log level display. For more information please 
 visit `--help` for any commands or sub-commands.
+Available Commands:
+  aws         Takes aws specific resource names from user
+  azure       Takes azure specific inputs from user
```

Comment:
Why the difference in `Takes aws specific resource names from user` vs `Takes azure specific inputs from user`?

What about something like:
```
   aws         Create resources in aws
   azure       Create resources in azure
```
`Resources` could be interchanged with `Windows node`